ci-operator/config/openshift/jaegertracing-jaeger-operator/openshift-jaegertracing-jaeger-operator-main__jaeger-product-ocp-4.19-lp-interop.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
+  cli:
+    name: "4.19"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+canonical_go_repository: github.com/jaegertracing/jaeger-operator
+images:
+- context_dir: .
+  dockerfile_path: tests/Dockerfile
+  to: jaeger-tests-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: jaeger-interop-aws
+  cron: 0 23 31 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "distributed-tracing-tests-*", "failure_type": "pod_failure", "classification": "Jaeger e2e tests", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "distributed-tracing-tests-*", "failure_type": "test_failure", "classification": "Jaeger e2e tests", "jira_project": "TRACING", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.19-lp","self-managed-lp","jaeger-lp"]'
+      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: rhn-support-ikanse
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      FIREWATCH_PRIVATE_DECK: "true"
+      OPERATORS: |
+        [
+            {"name": "elasticsearch-operator", "source": "redhat-operators", "channel": "stable-5.8", "install_namespace": "openshift-operators-redhat","target_namespace": "", "operator_group": "openshift-operators-redhat"},
+            {"name": "amq-streams", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators", "target_namespace": ""},
+            {"name": "jaeger-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-distributed-tracing","target_namespace": "", "operator_group": "openshift-distributed-tracing"},
+            {"name": "opentelemetry-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-opentelemetry-operator", "target_namespace": "", "operator_group": "openshift-opentelemetry-operator"}
+        ]
+      USER_TAGS: |
+        scenario jaeger
+    test:
+    - ref: install-operators
+    - ref: distributed-tracing-tests-jaeger
+    workflow: firewatch-ipi-aws
+- as: jaeger-interop-aws-fips
```

Comment:
Jaeger is not FIPS compliant and we will not be adding FIPS support for it as its deprecated and will be removed from the product end of this year.